draft-ietf-ice-rfc5245bis.xml

```diff
@@ -372,16 +372,15 @@ STUN response ->            /  check
 It is important to note that the STUN requests are sent to and from
 the exact same IP addresses and ports that will be used for media
 (e.g., RTP and RTCP). Consequently, agents demultiplex STUN and
-RTP/RTCP using contents of the packets, rather than the port on which
-they are received. Fortunately, this demultiplexing is easy to do,
-especially for RTP and RTCP.
+media using the contents of the packets, rather than the port on which
```

Comment:
This misses data channels.

Reply:
"media" here includes data channels.  But I agree it could be more explicit.  Should I throw in "SCTP" with the "e.g, RTP and RTCP" above?  Then we would need to add a reference.